### ISSUE TITLE
aria-pressed accessiblity issue

### ISSUE THREAD

**Issue by kmurphychi247** [stamat/youtube-background#72]
First of all thank you so much for this js library.  We have this set up but it is failing lighthouse tests at the moment.  The buttons were updated to use aria-pressed in this most recent release but according to accessibility rules, aria-pressed shouldn't be used for Play and Pause buttons.  They should be aria-checked with the role=“switch”
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed

**Comment by stamat** [comment]
Hey there @kmurphychi247 so sorry it took me full 2 years to reply 😢 Play and Pause buttons were experimental.

The a11y issue has been resolved in the v1.2.0. Thank you so much for reporting it! 🙏
